.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: bmbJJhTYflmvp4musILhQ4vscjSi794RE
```

Comment:
Hey Rita, we really do not need the repo_token since we're using Travis-CI.

Reply:
Alright. I just pushed the changes